galley/pkg/config/analysis/diag/message_test.go

```diff
@@ -67,6 +67,6 @@ func TestMessage_JSON(t *testing.T) {
 	m := NewMessage(mt, &resource.Instance{Origin: testOrigin{name: "toppings/cheese", ref: testReference{"path/to/file"}}}, "Feta")
 
 	j, _ := json.Marshal(&m)
-	g.Expect(string(j)).To(Equal(`{"code":"IST-0042","documentation_url":"https://istio.io/docs/reference/config/analysis/IST-0042"` +
+	g.Expect(string(j)).To(Equal(`{"code":"IST-0042","documentation_url":"https://istio.io/docs/reference/config/analysis/ist-0042/"` +
```

Comment:
What is this? There doesn't seem to be a code IST-0042 in the docs. Also all the other ones have no dash, i.e., they are all of the form `analysis/istNNNN`

Reply:
Oh, I see it's just a unit test ... might want to take out the `-` anyway.

Reply:
Fixed